hddfancontrol/__init__.py

```diff
@@ -759,19 +761,57 @@ def setSpeed(self, target_prct: int) -> None:
     def setPwmValue(self, value: int) -> None:
         """Set fan PWM value."""
         assert 0 <= value <= 255
+        self.setEnabledValue(1)
+        self.logger.debug(f"Setting PWM value to {value}")
+        with open(self.pwm_filepath, "wt") as pwm_file:
+            pwm_file.write(str(value))
+
+    def getEnabledValue(self) -> Optional[int]:
+        """Read the enabled value of the fan."""
+        if self.enable_filepath is not None:
+            try:
+                with open(self.enable_filepath, "rt") as enable_file:
+                    enabled_val = int(enable_file.read().strip())
+                    self.logger.debug(f"Fan is in enabled state {enabled_val}")
+                    return enabled_val
+            except FileNotFoundError:
```

Comment:
I don't think it is needed to handle this exception.
If `FileNotFoundError` is raised here, it means the file just disappeared while hddfancontrol was running, and this is so unexpected that I'd rather crash than continue.

Reply:
I thought this is needed to address issue #36. This exception is also handled in https://github.com/desbma/hddfancontrol/blob/f57d8b6775e5dd170a014044b78c5fa04f158f49/hddfancontrol/__init__.py#L763-L771

I think `getEnabledValue` is called earlier than `setPwmValue` so this exception should be thrown to not make a breaking change for pwm devices without enable.

What do you think?

Reply:
> I think getEnabledValue is called earlier than setPwmValue

I missed that, it's all good then.